backend/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.11-slim
+
+RUN apt-get update && \
+    apt-get install -y
+
+COPY requirements.txt .
+
+RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
Could you please create a python environment to avoid the console warning ?

Reply:
I did something slightly different to get rid of this warning: I created and used a user `gunicorn` to run the commands so that we do not use root.